nslsii/sync_experiment/sync_experiment.py

```diff
@@ -88,21 +88,31 @@ def validate_proposal(data_session_value, beamline) -> Dict[str, Any]:
 
 def authenticate(username):
 
-    auth_server = Server("dc2.bnl.gov", use_ssl=True)
+    authenticated = False
+    for server in ["1", "2", "3"]:
+        if authenticated:
+            break
+        auth_server = Server(f"dc{server}.bnl.gov", use_ssl=True)
+
+        try:
+            connection = Connection(
+                auth_server,
+                user=f"BNL\\{username}",
+                password=getpass("Password : "),
```

Comment:
Wouldn't it ask for the password (at most) three times?

Reply:
lol, just noticed it's merged already :)

Reply:
In the case of unable to connect, it never gets to the password prompt, just fails. So you should just get one password prompt no matter what